cdap-docs/admin-manual/source/security.rst

```diff
@@ -456,13 +450,11 @@ security components are working as expected:
 
     {"auth_uri":["http://localhost:10009/token"]}
 
-- Submit a username and password to one of the URIs to obtain an ``AccessToken``:
+- Submit a username and password to one of the URIs to obtain an ``AccessToken``::
 
-  .. tabbed-parsed-literal::
-	
-    curl -v -w"\n" -X GET -u <username>:<password> <auth-uri>
+    GET -u <username>:<password> <auth-uri>
```

Comment:
Isn't it more correct to say to send Basic Authorization Headers? `-u` is a `curl` thing...